src/gen/js/service.js.template

```diff
@@ -258,7 +258,10 @@ function formatServiceError(response, data, options) {
 }
 
 function processError(req, error) {
-	return Promise.resolve({ res: { raw: {}, data: error, error: true } });
+  const { processError } = options;;
+  const res = { res: { raw: error, data: error, error: true } };;
```

Comment:
Changing raw (which is indented for the raw Fetch response object) to an error object could be a breaking change. Can you change that back to `{}`?